### DISCUSSION TITLE
How to create a coil through comsol

### DISCUSSION THREAD

**Discussion by 784622315** [simnibs/simnibs#370] [2. General]
Hello, I saw a document about how to generate the SIMNIBS format from the coil model created by COMSOL, but it was very simple. Could you explain the process in detail?
![image](https://github.com/simnibs/simnibs/assets/148027353/46b5f3c3-848e-4df0-bc9a-caad82a9bc79)

**Comment by bjornsig** [comment]
If you have the coil model in comsol, you have to export the magnetic vector potential on a regular grid and use custom code to convert it to a nifti. If you don't know how to do this, consider asking the COMSOL support or in their forum. We are sorry we can't help you more as this is a COMSOL problem.

**Reply by 784622315** [reply]
OK, thank you very much for your answer.
